braindecode/augmentation/functional.py

```diff
@@ -144,7 +250,7 @@ def fft_surrogate(X, y, magnitude, random_state, *args, **kwargs):
     return transformed_X, y
 
 
-def _pick_channels_randomly(X, magnitude, random_state):
+def _pick_channels_randomly(X, magnitude, random_state=None):
```

Comment:
I tend to avoid default params in private aux functions to avoid forgetting to pass the param